Gemfile

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+gem 'rubocop', '~> 1.18.4', require: false
+
+gem 'fasterer', '~> 0.9.3'
```

Comment:
Where we can find `fasterer 0.9.3`?
[Here](https://rubygems.org/gems/fasterer) the latest version is `0.9.0`

Reply:
fixed